packages/mission-center/mission-center.pacscript

```diff
@@ -0,0 +1,47 @@
+pkgname="mission-center"
+pkgver="0.5.2"
+pkgrel="1"
```

Comment:
```suggestion
```
`pkgrel` is optional in pacscripts, it will be 1 by default anyway

Reply:
i didn't remember i add that.
but eh, sure.